src/main/scala/vectorpipe/vectortile/Pipeline.scala

```diff
@@ -1,151 +1,194 @@
- * into a set of vector tiles and how to pass information to subsequent zoom
- * levels.
- *
- * [It is worth noting that nothing in this interface restricts the input data
- * to having originated from OSM.  Any DataFrame containing a column of JTS
- * geometry is a valid target.]
- *
- * This API treats vector tile generation as an inherently hierarchical process
- * governed by a series of operations.  Every zoom level will be addressed in
- * the same way, and the sequence of operations can be depicted schematically as
- * below:
- *
- *     ┌──────────┐
- *     │ Zoom z-1 │
- *     └──────────┘
- *          ↑
- *       simplify
- *          ↑
- *        reduce
- *          ↑
- *     ╔══════════╗                          ┌─────────────┐
- *     ║  Zoom z  ║ → select → clip → pack → │ VECTOR TILE │
- *     ╚══════════╝                          └─────────────┘
- *
- * Each of these operations is documented below.  The only extra note is that [[
- * reduce]] and [[simplify]] will be called prior to processing the initial zoom
- * level.
- */
+  * The interface governing the transformation from processed OSM dataframes to
+  * vector tiles.
+  *
+  * After loading an OSM source and transforming via
+  * [[vectorpipe.OSM#toGeometry OSM.toGeometry]] and any filtering/modification
+  * steps, one may want to output a set of vector tiles from the remaining
+  * geometry and metadata.  The vector tile output process is governed by a
+  * Pipeline definition that describes how to convert a data frame into a set of
+  * vector tiles and how to pass information to subsequent zoom levels.
+  *
+  * [It is worth noting that nothing in this interface restricts the input data
+  * to having originated from OSM.  Any DataFrame containing a column of JTS
+  * geometry is a valid target.]
+  *
+  * This API treats vector tile generation as an inherently hierarchical process
+  * governed by a series of operations.  Every zoom level will be addressed in
+  * the same way, and the sequence of operations can be depicted schematically as
+  * below:
+  * {{{
+  *     ┌──────────┐
+  *     │ Zoom z-1 │
+  *     └──────────┘
+  *          ↑
+  *       simplify
+  *          ↑
+  *        reduce
+  *          ↑
+  *     ╔══════════╗                          ┌─────────────┐
+  *     ║  Zoom z  ║ → select → clip → pack → │ VECTOR TILE │
+  *     ╚══════════╝                          └─────────────┘
+  * }}}
+  * Each of these operations is documented below.
+  * The only extra note is that [[vectorpipe.vectortile.Pipeline#reduce reduce]]
+  * and [[vectorpipe.vectortile.Pipeline#simplify simplify]] will be called prior
+  * to processing the initial zoom level.
+  */
 trait Pipeline {
-  /**
-   * The root URI for output.
-   */
-  val baseOutputURI: java.net.URI
 
   /**
-   * Name of the column containing the JTS geometry.
-   */
-  val geometryColumn: String
+    * Name of the column containing the JTS geometry.
+    */
+  def geometryColumn: String
```

Comment:
I changed `geometryColumn` and `layerMultiplicity` to `def` from `val` to give users more freedom in how they are defined since generally traits shouldn't define abstract vals.  Also changed to make the scaladoc links work properly :p